tests/api.treatments.test.js

```diff
@@ -60,6 +61,43 @@ describe('Treatment API', function ( ) {
     });
   });
 
+
+  it('post single treatments in zoned time format', function (done) {
+   
+    var current_time = Date.now();
+    console.log('Testing date with local format: ', _moment(current_time).format("YYYY-MM-DDTHH:mm:ss.SSSZZ"));
```

Comment:
use `moment.toISOString()` instead of the format, in a few places

Reply:
@jasoncalabrese Note the formats are intentionally different to test inserting a zoned time, so changing those to toISOString() would break the test